### PR TITLE
add project authors file

### PR DESCRIPTION
**Pull request by loriab**
## Description

* It's nice to collect code author info so we have a chance of finding and addressing each other by name, not GitHub ID, at conferences. [Codemeta](https://codemeta.github.io/) is a way to do this.

* @dgasmith @alenaizan @doaa-altarawy @taylor-a-barnes @loriab @nuwandesilva @dotsdl @awvwgk @zachglick @AlexHeide @jhrmnn @ahurta92 @psi-rking @jygrace @sjrl @ffangliu @vivacebelles @devinamatthews @JonathonMisiewicz @Lnaden @jeffschriber @CDSherrill @eljost @WardLT @mattwelborn @twindus, you have been identified as a substantial code author or otherwise contributor to QCEngine or to QCDB code that migrated here.
  - [ ] If you don't want to be in this file, let me know. **Please opt-in** by responding in a comment
  - [ ] Please look over your entry and let me know any corrections, affiliation, accents, whether the middle initial slot needs deleting, etc.
  - [ ] Please let me know your [ORCID](https://orcid.org/). If you don't have one, if you're spending more than an undergrad summer in academia, it's probably worth registering for one.

* @jygrace @sjrl @dotsdl @eljost @ahurta92 @jhrmnn @ffangliu @taylor-a-barnes, I don't have email addresses for you.
  - [x] Email addresses won't go in this file on GH of course, but please convey them to me by GH (obfuscated), slack, or email (gmail `lori.burns`).

* EDIT: If you're not in this file and you think you should be, you could be right. Contact me.

* EDIT: checklist
  - [x] dgasmith
  - [x] alenaizan
  - [x] doaa-altarawy
  - [x] taylor-a-barnes
  - [x] loriab
  - [x] nuwandesilva
  - [x] dotsdl
  - [x] awvwgk
  - [x] zachglick
  - [x] AlexHeide
  - [x] jhrmnn
  - [x] ahurta92
  - [x] psi-rking
  - [x] jygrace
  - [x] sjrl
  - [x] ffangliu
  - [x] vivacebelles
  - [x] devinamatthews
  - [x] JonathonMisiewicz
  - [x] Lnaden
  - [x] jeffschriber
  - [x] CDSherrill
  - [x] eljost
  - [x] WardLT
  - [x] mattwelborn
  - [x] twindus
  - [x] MSG
  - [ ] JFS
  - [x] j-wags
  - [x] farhadrgh
  - [x] jthorton

## Status

- [ ] Code base linted
- [ ] Ready to go

### CODE REVIEW

**Comment by devinamatthews** [comment]
@loriab can you remind me what I contributed? I can't think of anything...

**Comment by loriab** [comment]
@devinamatthews You're right on nothing directly, but you gave me code that helped me write https://github.com/qcdb/qcdb/blob/master/qcdb/programs/cfour/jajo.py and similar. So you're welcome to choose to be associated with both, none, or either of the QCEngine and QCDB projects. This PR is basically the information gathering stage, so we can send out email.

File: codemeta.json [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+            "@id": "https://orcid.org/XXXX",
+            "identifier": "https://github.com/AlexHeide"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Jan",
+            "additionalName": "XXXX.",
+            "familyName": "Hermann",
+            "affiliation": "XXXX",
+            "@id": "https://orcid.org/XXXX",
```
Comment (jhrmnn) [inline codemeta.json:112]:
```suggestion
            "familyName": "Hermann",
            "affiliation": "Free University of Berlin",
            "@id": "https://orcid.org/0000-0002-2779-0749",
```

Comment (jhrmnn) [reply codemeta.json:112]: I opt in to be in the file. Please use my public Github email.

**Comment by devinamatthews** [comment]
Oh that, yes please include me damatthews@smu.edu.

File: codemeta.json [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+            "identifier": "https://github.com/dotsdl"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Sebastian",
+            "familyName": "Ehlert",
+            "affiliation": "Mulliken Center for Theoretical Chemistry, Institut f\u0188r Physikalische und Theoretische Chemie, Universit\u0164t Bonn",
+            "@id": "https://orcid.org/0000-0001-7809-771X",
+            "identifier": "https://github.com/awvwgk"
+        },
```
Comment (awvwgk) [inline codemeta.json:78]: Happy to be listed here.

Unless unicode escape sequences are preferred over actual unicode I would suggest:
```suggestion
        {
            "@type": "Person",
            "givenName": "Sebastian",
            "familyName": "Ehlert",
            "affiliation": "Mulliken Center for Theoretical Chemistry, Institut für Physikalische und Theoretische Chemie, Universität Bonn",
            "@id": "https://orcid.org/0000-0001-7809-771X",
            "identifier": "https://github.com/awvwgk"
        },
```

**Comment by CDSherrill** [comment]
Looks ok!

File: codemeta.json (hunk 2 above)
Comment (loriab) [reply codemeta.json:78]: I can't remember why I used unicode escape sequences for the Psi4 one of these. But until some validator or scraper protests, I'm glad to go with the actual characters.

**Comment by jeffschriber** [comment]
Looks good, thanks!

**Comment by JonathonMisiewicz** [comment]
If I'm in here for findif driver, LGTM. If not, could you give me a refresher on what my part was?

**Comment by loriab** [comment]
@JonathonMisiewicz, yes, findif procedure.

**Comment by WardLT** [comment]
Thanks, @loriab! My ORCID is 0000-0002-1323-5939 and email lward@anl.gov

**Comment by dgasmith** [comment]
Thanks for doing this @loriab! This will be a great addition to the project.

**Comment by dgasmith** [comment]
We may want to consider the following:
 - @jchodera and @j-wags for contributing MM-related technical expertise and advice.
 - @farhadrgh for ANI integration

**Comment by vivacebelles** [comment]
Everything looks good on my end @loriab, my ORCID is 0000-0002-1686-778X and email <alolinco@iastate.edu>.

**Comment by alenaizan** [comment]
Looks good.

**Comment by eljost** [comment]
Looks good to me. I sent you everything via mail.

**Comment by ahurta92** [comment]
Thanks, @loriab! Looks good.  My email is ahurta92@gmail.com

**Comment by mattwelborn** [comment]
Thanks @loriab!

```
        {
            "@type": "Person",
            "givenName": "Matthew",
            "familyName": "Welborn",
            "affiliation": "Entos, inc.",
            "identifier": "https://github.com/mattwelborn"
        },
```

**Comment by loriab** [comment]
@ahurta92 confirm no middle name and no ORCID?

**Comment by twindus** [comment]
Thanks @loriab!  My orcid is https://orcid.org/0000-0001-6065-3167.  Everything else looks fine.

**Comment by dotsdl** [comment]
Thanks @loriab! Here is my info:

```
{
            "@type": "Person",
            "givenName": "David",
            "additionalName": "L.",
            "familyName": "Dotson",
            "affiliation": "Open Force Field Initiative",
            "@id": "https://orcid.org/0000-0001-5879-2942",
            "identifier": "https://github.com/dotsdl"
        },
```

**Comment by sjrl** [comment]
Thanks @loriab! My orcid is https://orcid.org/0000-0001-7006-9378

**Comment by zachglick** [comment]
Looks good, thank you Lori

**Comment by taylor-a-barnes** [comment]
Thanks @loriab!  My ORCID is https://orcid.org/0000-0001-9396-1094.

**Comment by jchodera** [comment]
This is awesome, folks! I don't need an acknowledgment at this time, but hope to contribute more substantially in the future!

**Comment by nuwandesilva** [comment]
Hi Lori,

My ORCID is  0000-0002-1443-5764

Thank you,
Nuwan

**Comment by farhadrgh** [comment]
Hi @loriab, Here is my info:

        {
            "@type": "Person",
            "givenName": "Farhad",
            "familyName": "Ramezanghorbani",
            "affiliation": "University of Florida",
            "@id": "http://orcid.org/0000-0002-7545-4416",
            "identifier": "https://github.com/farhadrgh"
        },

**Comment by psi-rking** [comment]
My entry looks fine, thanks.  -Rollin

**Comment by loriab** [comment]
@j-wags, please post your info if you'd like to be included in the codemeta.

**Comment by AlexHeide** [comment]
My info:
```
{
    "@type": "Person",
    "givenName": "Alex",
    "additionalName": "G.",
    "familyName": "Heide",
    "affiliation": "Center for Computational Quantum Chemistry, University of Georgia",
    "@id": "https://orcid.org/0000-0002-9780-8123",
    "identifier": "https://github.com/AlexHeide"
},
```

**Comment by ahurta92** [comment]
> @ahurta92 confirm no middle name and no ORCID?

oh sorry,  middle name is Guillermo and my ORCID is 0000-0002-2460-8059.
Thank you!

**Comment by doaa-altarawy** [comment]
My ORCID is
0000-0002-7795-4422
daltarawy@vt.edu

**Comment by j-wags** [comment]
Sorry for the delay @loriab, and thanks for nominating me, @dgasmith!

```
        {
            "@type": "Person",
            "givenName": "Jeffrey",
            "additionalName": "R.",
            "familyName": "Wagner",
            "affiliation": "Open Force Field Initiative",
            "@id": "https://orcid.org/0000-0001-6448-0873",
            "identifier": "https://github.com/j-wags"
        },
```

**Comment by Lnaden** [comment]
{
            "@type": "Person",
            "givenName": "Levi",
            "additionalName": "N.",
            "familyName": "Naden",
            "affiliation": "Molecular Sciences Software Institute",
            "@id": "https://orcid.org/0000-0002-3692-5027",
            "identifier": "https://github.com/lnaden"
        },

**Comment by ffangliu** [comment]
Hi Lori,

Here is my information.

   {
        "@type": "Person",
        "givenName": "Fang”,
        "additionalName": "",
        "familyName": "Liu",
        "affiliation": “Emory University”,
        "@id": "https://orcid.org/0000-0003-1322-4997",
        "identifier": "https://github.com/ffangliu"
    },

**Comment by dgasmith** [comment]
This is really amazing @loriab! Thank you for putting the time and effort into this.

**Pull request merged**